pkg/web/api/v1/user.go

```diff
@@ -13,9 +13,9 @@ type User struct {
 	ID        ulid.ULID  `json:"id,omitempty"`
 	Name      string     `json:"name"`
 	Email     string     `json:"email"`
-	Passsword string     `json:"password,omitempty"`
+	Password  string     `json:"password,omitempty"`
 	Role      string     `json:"role"`
-	LastLogin *time.Time `json:"last_login"`
+	LastLogin *time.Time `json:"last_login,omitempty"`
```

Comment:
I added `omitempty` here after noticing it wasn't included, but it can be removed.

Reply:
This is good - I think it should be `omitempty` here.